build.gradle

```diff
@@ -16,6 +16,9 @@ project.configurations {
 }
 
 project.dependencies {
+
+    implementation "org.xerial:sqlite-jdbc:3.7.2"
```

Comment:
Is there any way to declare this dependency in the TargetedMS module's test code, or does it have to be here in the central test harness? If it has to be here, please add a comment as to its use case.

Reply:
It did not work with the dependency declaration mentioned in targettedMS module(since test is part of that module).
I am not aware of other way, let me know if there is another?